app/src/main/res/values/strings.xml

```diff
@@ -1794,4 +1794,41 @@
     <string name="recommended_content_survey_dialog_submitted_message">Feedback submitted. Thank you!</string>
     <!-- /Recommended Content -->
 
+    <!-- Contributions Dashboard -->
+    <string name="donor_history_title">Donor history</string>
+    <string name="donor_history_save_button">Save</string>
+    <string name="donor_history_update_donor_status_default">Update donor status</string>
+    <string name="donor_history_update_donor_status_donor">Donor</string>
+    <string name="donor_history_update_donor_status_not_a_donor">Not a donor</string>
+    <string name="donor_history_last_donated">Last donated</string>
+    <string name="donor_history_last_donated_hint">Add date of last donation</string>
+    <string name="donor_history_last_donated_date_picker_title">Select date</string>
+    <string name="donor_history_last_donated_date_picker_label">Enter date</string>
+    <string name="donor_history_last_donated_date_picker_cancel">Cancel</string>
+    <string name="donor_history_last_donated_date_picker_ok">OK</string>
```

Comment:
Can we use `android.R.string.ok` and `android.R.string.cancel`, especially for such an experimental feature?

Reply:
Yes, that makes sense to me.